modules/ROOT/pages/browser-support.adoc

```diff
@@ -0,0 +1,26 @@
+= Browser Support for Anypoint Platform
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+// page-aliases: 
+
+Use Anypoint Platform with the latest version of any supported browser:
+
+* Microsoft Edge with Chromium
+* Microsoft Edge wihtout Chromium (supported through December 31, 2020)
+* Google Chrome
+* Mozilla Firefox
+* Safari
+
+The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
+
+== Unsupported Browsers
+
+Internet Explorer 11 is not supported by MuleSoft.
+While the browser may work with Anypoint Platform, you may experience issues.
+
+== See Also
+
+https://help.salesforce.com/articleView?id=getstart_browsers_sfx.htm&type=5[Supported Browsers for Lightning Experience^]
```

Comment:
What's the little ^ doohickey for? I don't recall seeing anything like this before....

Reply:
It's for launching a new window instead of opening in the same window. We haven't gotten all this confirmed or documented yet in style guide, it's in process.